client/src/app/tabs/cloud-dmn/DmnEditor.js

```diff
@@ -422,6 +468,15 @@ export class DmnEditor extends CachedComponent {
       });
     }
 
+    try {
+      const engineProfile = this.engineProfile.get();
+
+      this.engineProfile.setCached(engineProfile);
+    } catch (err) {
+
+      // TODO
```

Comment:
??

Reply:
This comes from https://github.com/camunda/camunda-modeler/blob/develop/client/src/app/tabs/bpmn/BpmnEditor.js#L453. Perhaps we can just mark it specifically that we don't need to react on error here.

Reply:
In fact, this doesn't happen as we don't allow unknown execution platform at this point.